core/src/test/java/com/sequenceiq/cloudbreak/service/cluster/RepositoryConfigValidationServiceTest.java

```diff
@@ -31,12 +35,16 @@
     @Mock(answer = Answers.RETURNS_DEEP_STUBS)
     private Client client;
 
+    private UrlAccessValidationService urlAccessValidationService;
+
     private RepositoryConfigValidationService underTest;
 
     @Before
     public void setUp() throws Exception {
-        initMocks(this);
         underTest = Mockito.spy(RepositoryConfigValidationService.class);
```

Comment:
We shouldn't spy anymore, because the package private method has been deleted from the tested class